examples/named/named_day_enum_marshal_bson.go

```diff
@@ -3,6 +3,9 @@ package day
 
 import (
 	"github.com/globalsign/mgo/bson"
+
+	mongo "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/bsontype"
```

Comment:
were now requiring users to add both drivers to their dependencies? Feels like this should be an option;

```
-bson=[ mgo | mongo-driver ]
```

Reply:
and if you want both?

Reply:
I build this support because I want both, I dont see a reason not to have both either

Reply:
Why would you need both database drivers? Do you have two connections to your DB using different drivers?